### PR TITLE
Add run date range selector

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: application/dash_application/utility/df_manipulation.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+
+import dash_core_components as core
+import dash_html_components as html
+import datetime
+import pandas
 from pandas import DataFrame
-from typing import List
+from typing import List, Tuple
 
+import pinery.column
```
[the remark this answers is not in the document]
Comment (apmasell) [reply application/dash_application/utility/df_manipulation.py]: Yes. Otherwise, this leads to the weird import problems we've seen.

**Comment by avarsava** [comment]
Doesn't seem like it displays correctly for me? Probably not be your fault though 
image

**Comment by apmasell** [comment]
I think it tries to keep the timespan when you adjust the start date, so if you adjust the end date first, it overwrites it.

**Comment by avarsava** [comment]
> I think it tries to keep the timespan when you adjust the start date, so if you adjust the end date first, it overwrites it.

Yeah that part's not clear, but I meant more about the edges of the calendar being cut off

**Comment by apmasell** [comment]
The problem seems to be the `day_size` attribute, which I've adjusted.

**Pull request merged**